util.c

```diff
@@ -583,17 +583,18 @@ int pad_file(struct image *image, const char *infile,
  * fallocate). This only uses methods that do not affect the offset of
  * fd.
  */
-static int write_zeroes(int fd, size_t size, off_t offset)
+static int write_zeroes(int fd, size_t size, off_t offset, unsigned char fillpattern)
```

Comment:
Perhaps update the comment above the function. Smth like "Write @size @fillpattern bytes at...", s/big-all-zero-buffer/big-buffer/, and end the second sentence there, then s/except that/For fillpattern=0, /.

Reply:
good point, fixed. I also updated the comment for `insert_image()`